lib/src/bridge/scanning_mixin.dart

```diff
@@ -1,42 +1,49 @@
 part of _internal;
 
 mixin ScanningMixin on FlutterBLE {
-  Stream<dynamic> _scanEvents;
+  Stream<ScanResult> _scanEvents;
 
   void _prepareScanEventsStream() {
     _scanEvents =
-        const EventChannel(ChannelName.scanningEvents).receiveBroadcastStream();
+        const EventChannel(ChannelName.scanningEvents).receiveBroadcastStream()
+            .handleError(
+              (errorJson) {
+            throw BleError.fromJson(jsonDecode(errorJson.details));
+          },
+          test: (error) => error is PlatformException,
+        ).map((scanResultJson) =>
+            ScanResult.fromJson(
+              jsonDecode(scanResultJson),
+              _manager,
+            ));
   }
 
-  Stream<ScanResult> startDeviceScan(
-    int scanMode,
-    int callbackType,
-    List<String> uuids,
-    bool allowDuplicates,
-  ) async* {
-    _methodChannel.invokeMethod(
-      MethodName.startDeviceScan,
-      <String, dynamic>{
-        ArgumentName.scanMode: scanMode,
-        ArgumentName.callbackType: callbackType,
-        ArgumentName.uuids: uuids,
-        ArgumentName.allowDuplicates: allowDuplicates,
-      },
-    );
-
+  Stream<ScanResult> startDeviceScan(int scanMode,
+      int callbackType,
+      List<String> uuids,
+      bool allowDuplicates,) {
```

Comment:
Run formatter, please

Reply:
I did. Unless the formatter is not automatically loaded on the project?